### PR TITLE
Implements multiple activities per methodology

### PR DESCRIPTION
**Pull request by isaaccodekill**
https://github.com/user-attachments/assets/b688e5c5-b88e-470d-b5b3-76e2b99e4410

includes ui changes and api changes

### CODE REVIEW

File: app/src/backend/formulas.ts [hunk 1]
```diff
@@ -216,9 +216,10 @@ export function handleIndustrialWasteWaterFormula(
   const degradableOrganicComponents = data["degradable-organic-components"];
   const methaneProductionCapacity =
     data["methane-production-capacity"] ?? DEFAULT_METHANE_PRODUCTION_CAPACITY; // TODO should this only be handled UI-side?
-  const removedSludge = data["removed-sludge"];
-  const methaneCorrectionFactor = data["methane-correction-factor"];
-  const methaneRecovered = data["methane-recovered"];
+  const removedSludge = data["total-organic-sludge-removed"];
+  const methaneCorrectionFactor = data["methane-correction-factor"]; // TODO @leminkoh is this correct? it doesn't exist on the ui
```
Comment (isaaccodekill) [inline app/src/backend/formulas.ts:220]: I don't if this is correct @lemilonkh  because this field doesn't exist on the UI

Comment (lemilonkh) [reply app/src/backend/formulas.ts:220]: This needs to come from the FormulaInput table (it's called MCF there
). Alternatively for now you can encode similar if/ else rules like we have for other formulas to not have to load this data until it's ready.

File: app/src/backend/formulas.ts [hunk 2]
```diff
@@ -244,8 +246,7 @@ export async function handleDomesticWasteWaterFormula(
   }
 
   const methaneProductionCapacity = DEFAULT_METHANE_PRODUCTION_CAPACITY; // TODO should this only be handled UI-side?
-  const removedSludge =
-    data["wastewater-inside-domestic-calculator-total-organic-sludge-removed"];
+  const removedSludge = data["total-organic-sludge-removed"];
```
Comment (lemilonkh) [inline app/src/backend/formulas.ts:249]: I think we might need to revert this change?
And need a better mapping of input IDs to actual formula values that allows the different names from different activities to be read here.

File: app/src/util/form-schema/manual-input-hierarchy.json [hunk 3]
```diff
@@ -717,20 +776,26 @@
             "prefills": [
               {
                 "key": "manufacturing-and-construction-type",
-                "value": "manufacturing-and-construction-type-institutional-buildings"
+                "value": "institutional-buildings"
```
Comment (lemilonkh) [inline app/src/util/form-schema/manual-input-hierarchy.json:779]: We will unfortunately have to revert the changes in this file.

**Review by lemilonkh — COMMENTED** [review]
Thanks :sparkles:

File: app/src/backend/formulas.ts (hunk 2 above)
Comment (isaaccodekill) [reply app/src/backend/formulas.ts:249]: this wasn't a change from the manual input. The formula was just referring to the wrong property

File: app/src/util/form-schema/manual-input-hierarchy.json (hunk 3 above)
Comment (lemilonkh) [reply app/src/util/form-schema/manual-input-hierarchy.json:779]: Seems to be thorough so let's go with it

File: app/src/backend/formulas.ts (hunk 1 above)
Comment (isaaccodekill) [reply app/src/backend/formulas.ts:220]: I'll extend this in a follow up ticket

Comment (lemilonkh) [reply app/src/backend/formulas.ts:220]: As long as it doesn't give us an error with saving for now that's good. Otherwise comment this line and replace it with a constant for now.

Comment (lemilonkh) [reply app/src/backend/formulas.ts:220]: Because the result will be NaN otherwise

**Review by lemilonkh — APPROVED** [review]
